multi_map_it_test.go

```diff
@@ -124,6 +124,31 @@ func TestMultiMap_Remove(t *testing.T) {
 	})
 }
 
+func TestMultiMap_RemoveEntry(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		targetValue := "value"
+		otherValue := "other value"
+		it.Must(m.PutAll(ctx, "key", targetValue, otherValue))
+		value := it.MustValue(m.Get(ctx, "key"))
+		assert.ElementsMatch(t, value, []interface{}{targetValue, otherValue})
+		// Remove only one of the values that corresponds to the key
+		ok, err := m.RemoveEntry(ctx, "key", targetValue)
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.True(t, ok)
+		remaining := it.MustValue(m.Get(ctx, "key"))
+		assert.Equal(t, []interface{}{otherValue}, remaining)
+		// Call should have no effect, expect "false" return value
```

Comment:
```suggestion
// Call should have no effect, expects "false" as a return value.
```

Reply:
Thanks, fixed to something similar https://github.com/hazelcast/hazelcast-go-client/pull/725/commits/b58fa1b6acabcb86878ab48d0b9fa24b5c27d21c